nj-core/src/convert.rs

```diff
@@ -176,6 +183,27 @@ impl JSValue<'_> for i64 {
     }
 }
 
+impl JSValue<'_> for u64 {
+
+    fn convert_to_rust(env: &JsEnv,js_value: napi_value) -> Result<Self, NjError> {
+        env.assert_type(js_value, crate::sys::napi_valuetype_napi_bigint)?;
+
+        let mut value: u64 = 0;
+        let mut lossy: bool = false;
```

Comment:
It would be ideal do something with this `lossy` boolean other than pass it in to be modified by the node runtime and never actually used. I'm not quite sure what though.

Reply:
Or we could create a wrapper for [BigInt](https://docs.rs/num-bigint/0.3.0/num_bigint/) and not just support 64 bits.  This way, developer has control over what level of abstraction they would prefer and dealing. of loss of precision

Reply:
After implementing `JSValue` and `TryIntoJSs` for `BigInt`, I think this function should just use `napi_get_value_bigint_words` and then we don't have to worry about any loss.

Reply:
I've removed this entirely in favor of wrapping `BigInt` and not supporting u64s. Though, I guess there's an argument to be made about adding the `TryIntoJs` implementation for `u64`s. I'll think about this more and maybe re-add it.

Reply:
I agree.  I support decision to remove them since semantics are not clear.   Eventually, we should support converting to u64  if this become frequent use case.